### DISCUSSION TITLE
My Introduction

### DISCUSSION THREAD

**Discussion by kushwaha001** [dipy/dipy#3118] [#introduceyourself]
Dear Mentors,

I hope this message finds you well. My name is Kanishk Kushwaha, and I am writing to express my interest in contributing to your organisation (DIPY) in GSOC 2024.

I have been following your organization's work for some time on your official youtube channel, and I am particularly interested in your projects. I am new to DIPY but will master it if I will get someone to guide me.

I am confident that I can bring value to your team and help further the goals of the project. I am eager to learn from the expertise within your team and contribute my own insights and ideas.

Thank you.

**Comment by pjsjongsung** [comment]
Hello! Note that you have to select a project to submit a proposal. Also, as a sub-organization of Python Software Foundation, we do need some contribution to our open source package. Starting by looking at the current issues might help.

Please let us know if you have any questions. Thank you!
